tests/backup/backup_restore_basic_test.go

```diff
@@ -2800,3 +2801,208 @@ var _ = Describe("{BackupRestoreOnDifferentK8sVersions}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+// ScheduleBackupDeleteAndRecreateNS Validates schedule backups when namespaces are deleted and recreated
+var _ = Describe("{ScheduleBackupDeleteAndRecreateNS}", func() {
+	var (
+		contexts                     []*scheduler.Context
+		cloudCredUID                 string
+		cloudCredName                string
+		backupLocationName           string
+		backupLocationUID            string
+		backupLocationMap            map[string]string
+		schedulePolicyName           string
+		schedulePolicyUid            string
+		scheduleName                 string
+		latestScheduleBackupName     string
+		restoreName                  string
+		appNamespaces                []string
+		scheduleNames                []string
+		backedUpNamespaces           []string
+		schedulePolicyintervalInMins int
+		numDeployments               int
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ScheduleBackupDeleteAndRecreateNS", "Verification of schedule backups when namespaces are deleted and recreated", nil, 59894)
+		numDeployments = 2
```

Comment:
It is advisable to verify that the length of the app-list is equal to 1 before setting numDeployments to 2. If the app-list contains two namespaces, each application will be deployed in two namespaces, which is unnecessary since a total of two namespaces will do the trick.

Reply:
Done.